docs/patterns/clustered-listening-tentacles.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Clustered Listening Tentacles
+description: Configuring the Octopus Tentacle in an active/passive cluster. 
+position: 0
+---
+
+You can configure a pair of Octopus Tentacles in an active/passive failover cluster on shared storage with the **Failover Cluster Manager**. You may need to do this if you're running an application in a failover cluster and would like to use Octopus Deploy to deploy your application to it irrespective of the fail-over state. In this scenario your Octopus Deploy Server will always be communicating with the Octopus Tentacle that is installed on the active node within the failover cluster. This approach has been tested on Windows Server 2016. 
+
+:::warning
+**Shared Storage Considerations**
+It is not possible to store the **tentacle.config** file in shared storage because the Tentacle.Certificate component gets encrypted using a node's machine-specific key. If you attempt to store the tentacle.config file in shared storage, you will encounter this error: `Key not vaild for use in specified state` upon switching to a new active node. This occurs because the new active node is not able to decrypt the private key resulting in the Tentacle service failing to start.
```

Comment:
* `tentacle.config` should be surrounded by backticks, not bold.
* typo in `vaild`